model/src/main/proto/topic.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+// Represents the play state of a single chapter.
+enum ChapterPlayState {
+  // The completion status is unknown.
+  COMPLETION_STATUS_UNSPECIFIED = 0;
+
+  // The chapter has not yet been started, but can be started by the player.
+  NOT_STARTED = 1;
+
```

Comment:
Are we not planning to keep a "In progress" kind of state for a chapter?

Reply:
Not yet. We've discussed in the past introducing persistent state for explorations, but this isn't yet supported in the backend or Oppia web, and is outside the scope of the prototype. It's also hard since we need to somehow represent the learner's placement in the state graph in a way that doesn't break with future versions of the exploration. Skill & story progress are easier since we just need to track completion stats, and the structures are simpler.

I think it would be a really valuable improvement to the UX, but may need to wait until post launch of the app.

/cc @seanlip